app/podcast/[id]/page.tsx

```
@@ -1,6 +1,8 @@
 // ABOUTME: Podcast episode detail page with player and JSON-LD
 // ABOUTME: Shows guest info, show notes, and platform badges
 import type { Metadata } from 'next';
+export const revalidate = 21600; // 6 hours ISR
+export const revalidate = 21600; // 6 hours ISR for podcast episodes
```

Biome (2.1.2)

[error] Shouldn't redeclare 'revalidate'. Consider to delete it or rename it.

'revalidate' is defined here:


(lint/suspicious/noRedeclare)